src/app/component/TierPrices/TierPrices.style.scss

```diff
@@ -0,0 +1,19 @@
+.TierPrices {
+    &-Item {
+        @include after-mobile {
+            padding-left: 5px;
+        }
+
+        @include mobile {
+            padding-left: 0;
+        }
+
+        &:last-child {
+            padding-bottom: 0;
+        }
+
+        &::before {
+            content: none;
+        }
+    }
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=4869008)